smtp.go

```diff
@@ -27,9 +27,17 @@ type Dialer struct {
 	// most cases since the authentication mechanism should use the STARTTLS
 	// extension instead.
 	SSL bool
-	// TSLConfig represents the TLS configuration used for the TLS (when the
+	// TLSConfig represents the TLS configuration used for the TLS (when the
 	// STARTTLS extension is used) or SSL connection.
 	TLSConfig *tls.Config
+	// RequireStartTLS represents the TLS security level required to
+	// communicate with the SMTP server.
+	//
+	// This defaults to OpportunisticStartTLS for backwards compatibility,
+	// but we recommend MandatoryStartTLS for all modern SMTP servers.
+	//
+	// This option has no effect if SSL is set to true.
+	RequireStartTLS StartTLSSecurity
```

Comment:
I think `StartTLSPolicy` would be a more fitting name for both `RequireStartTLS` and `StartTLSSecurity`.

Reply:
Fixed in 0c066ca.